### PR TITLE
Raise the visibility of other networks and methods of interaction

### PR DESCRIPTION
**Pull request by saharct**
Changes since [PR 6947 ](https://github.com/aptos-labs/aptos-core/pull/6947) to address concerns raised in [issue 7067](https://github.com/aptos-labs/aptos-core/issues/7067):
- Reformatted the first section to increase the visibility of the other networks and methods of interacting with them
- Mentioned local testnet and hyperlinked the relevant page for developing locally
- Rephrased the mainnet portion to more explicitly state our position on funding

### CODE REVIEW

**Review by clay-aptos — APPROVED** [review]
Nice. Thanks, Christian!

File: developer-docs-site/docs/guides/get-test-funds.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ slug: "get-test-funds"
...
-
-Other than funding your account in mainnet, the rest of your workflow remains the same as in devnet and testnet. Just remember, in mainnet you are working with real, monetary value that cannot simply be recreated.
+Other [Aptos networks](./system-integrators-guide.md#choose-a-network) and methods of interaction:
+  * **Devnet** - To see how to interact with the Aptos Faucet in software, conduct [your first transaction](../tutorials/first-transaction.md) in devnet. 
+  * **Local testnet** - Start a local testnet with a faucet to [develop locally](../nodes/local-testnet/index.md) with your preferred method. 
+  * **Mainnet** - There is no faucet available in mainnet to fund your account, so the method for obtaining the Aptos APT tokens is beyond the scope of this site and entirely up to you. Apart from the difference in the funding process, the rest of your workflow remains the same as in devnet and testnet. Just remember, in mainnet you are working with real, monetary value that cannot simply be recreated.
```
Comment (davidiw) [inline developer-docs-site/docs/guides/get-test-funds.md:15]:
```suggestion
  * **Mainnet** - There is no faucet available in mainnet to fund your account, so the method for obtaining the Aptos APT tokens is beyond the scope of this site and entirely up to you. Apart from the difference in the funding process, the rest of your workflow remains the same as in devnet and testnet.
```

Comment (davidiw) [reply developer-docs-site/docs/guides/get-test-funds.md:15]: let's circle back for the right approach to expressing this

**Review by davidiw — COMMENTED** [review]
looks good

**Review by davidiw — APPROVED** [review]

File: developer-docs-site/docs/guides/get-test-funds.md (hunk 1 above)
Comment (saharct) [reply developer-docs-site/docs/guides/get-test-funds.md]: Sounds good!

**Pull request merged**